sdk/core/Azure.Core/src/Pipeline/HttpPipelineSynchronousPolicy.cs

```diff
@@ -6,24 +6,37 @@
 
 namespace Azure.Core.Pipeline
 {
+    /// <summary>
+    /// Represents a <see cref="HttpPipelinePolicy"/> that doesn't do any asynchronous or blocking operations.
```

Comment:
This reads odd. Maybe "...do any asynchronous operations or block concurrent threads."?